documentation/modules/overview/con-kafka-concepts-components.adoc

```diff
@@ -14,7 +14,7 @@ Each of the other Kafka components interact with the Kafka cluster to perform sp
 
 .Kafka component interaction
 
-image:overview/kafka-concepts-supporting-components.png[Kafka components]
+image:overview/kafka-concepts-supporting-components.png[Data flows between several Kafka components and the Kafka cluster]
```

Comment:
Can we name the components so it's more descriptive?

Reply:
That might make the alt-text a bit lengthy. Based on guidelines, I've added a signpost:

"See the component descriptions after this image."